ratpack-core/src/test/groovy/ratpack/http/client/HttpClientRedirectionSpec.groovy

```diff
@@ -53,6 +53,73 @@ class HttpClientRedirectionSpec extends BaseHttpClientSpec {
     pooled << [true, false]
   }
 
+  def "can follow redirect get request with query parameters"() {
```

Comment:
Added just to be sure that the changes I made for the absolute URI didn't break anything, old code also passed this test.

Reply:
:+1: